src/megatron/bridge/recipes/llama/llama3.py

```diff
@@ -192,6 +193,46 @@ def llama3_8b_128k_pretrain_config(**user_kwargs: Unpack[Llama3CommonKwargs]) ->
     return _llama3_common(**combined_kwargs)
 
 
+def llama3_8b_low_precision_pretrain_config(
+    mixed_precision_recipe: str, **user_kwargs: Unpack[Llama3CommonKwargs]
+) -> ConfigContainer:
+    """Return a low precision (FP8 Current Scaling/MXFP8/NVFP4) pre-training config for Llama 3 8B.
+
+    Args:
+        mixed_precision_recipe (str): The mixed precision recipe to use. Valid options are:
+            - "bf16_with_mxfp8_mixed"
+            - "bf16_with_fp8_current_scaling_mixed"
+            - "bf16_with_nvfp4_mixed"
+        user_kwargs (Unpack[Llama3CommonKwargs]): Additional user-specified configuration options.
+
+    Returns:
+        ConfigContainer: The pre-training configuration for Llama 3 8B.
+
+    See `_llama3_common` for the full list of parameters.
+    """
+    assert mixed_precision_recipe in [
+        "bf16_with_mxfp8_mixed",
+        "bf16_with_fp8_current_scaling_mixed",
+        "bf16_with_nvfp4_mixed",
+    ], f"Invalid low precision recipe: {mixed_precision_recipe}"
+    precision_config = get_mixed_precision_config(mixed_precision_recipe)
+    recommended_kwargs: Llama3CommonKwargs = {
+        "hf_path": "meta-llama/Meta-Llama-3-8B",
+        "tensor_parallelism": 1,
+        "pipeline_parallelism": 1,
+        "context_parallelism": 2,
+        "sequence_parallelism": False,
```

Comment:
these were updated in https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/1199 so we use the megatron core terminology everywhere

```suggestion
        "tensor_model_parallel_size": 1,
        "pipeline_model_parallel_size": 1,
        "context_parallel_size": 2,
        "sequence_parallel": False,
```